h3/library/Makefile

```diff
@@ -2,14 +2,25 @@ TARGET = ../h3.js
 AUXLIB = h3-js.umd.js
 H3_JS_VERSION ?= 3.7.0
 
+ESLINT ?= ./node_modules/eslint/bin/eslint.js
 NPM ?= npm
-CP ?= cp
+CAT ?= cat
 
+JS_FILES = \
+	$(AUXLIB) \
+	h3_version.js
+
+LINT_FILES = \
+	h3_version.js
 
 .PHONY: all
 all: ${TARGET}
-$(TARGET): $(AUXLIB)
-	$(CP) $(AUXLIB) $(TARGET)
+$(TARGET): $(JS_FILES)
+	rm -f $(TARGET)
+	touch $(TARGET)
```

Comment:
Why touch after remove?

Reply:
No reason at all. I would say that was added to create back the file. But I see it's not needed. I've just removed it from all the modules.